root/etc/rc.d/rc.local

```diff
@@ -5,7 +5,14 @@ exportKCL
 
 # setup root password
 if [[ ! -z "$KCL_FDI_ROOTPW" ]]; then
-  echo "root:$KCL_FDI_ROOTPW" | chpasswd >/dev/null
+  # remove possible double and/or single quotes surrounding password string
+  KCL_FDI_ROOTPW=`echo $KCL_FDI_ROOTPW | sed "s/\('\|\"\)//g"`
```

Comment:
This is to increase foolproof of this code so it it does not break even if a user puts the password in quotes, so the following forms of passwords in pxeconfig will just work w/o issues:
fdi.rootpw="$1$_redhat_$i3.3Eg7ko/Peu/7Q/1.wJ/"
fdi.rootpw='$1$_redhat_$i3.3Eg7ko/Peu/7Q/1.wJ/'
fdi.rootpw=$1$_redhat_$i3.3Eg7ko/Peu/7Q/1.wJ/
fdi.rootpw=redhat

What is your concern having this here?